README.md

```diff
@@ -26,6 +26,12 @@ git clone https://github.com/vlang/sdl.git ~/.vmodules
 cd ~/.vmodules/sdl
 git checkout 2.0.12
 ```
+and for Windows (and then following the needed steps mentioned in the Windows specific section)
+```bash
+git clone https://github.com/vlang/sdl.git %HOMEPATH%/.vmodules/sdl
```

Comment:
This is what `v install sdl` does... might be better to stick with the standard command.

Reply:
I think I don't really understand what you're getting at. If it does this on Unix, why was it described?
It doesn't work this way on Windows, tested on Windows 7 and Windows 10.
There, a "v install sdl" only clones the main branch, other branches are not available.